File: Topic Based Problems/01 Fundamentals/1 Beginner/12 Power Calculation/C++/PowerCalculation.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main() {
    double base, exponent;

    cout << "Enter base: ";
    cin >> base;

    cout << "Enter exponent: ";
    cin >> exponent;

    cout << "Power of " << base << " to the " << exponent << " is: " << fixed << setprecision(2) << pow(base, exponent) << endl;

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/12 Power Calculation/Java/PowerCalculation.java
import java.util.Scanner;

public class PowerCalculation {
    public static void main(String[] args) {
        double base, exponent;

        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter base: ");
        base = scanner.nextDouble();

        System.out.print("Enter exponent: ");
        exponent = scanner.nextDouble();

        System.out.printf("Power of %.2f to the %.2f is: %.2f\n", base, exponent, Math.pow(base, exponent));

        scanner.close();
    }
}
